components/integrations_marketplace/marketplace_item/marketplace_item.jsx

```diff
@@ -140,12 +141,24 @@ export default class MarketplaceItem extends React.Component {
             const ariaLabel = `${this.props.name}, ${this.props.description}`.toLowerCase();
             const versionLabel = `(${this.props.version})`;
 
+            let pluginIcon;
+            if (this.props.iconData) {
```

Comment:
i know we do this in other places, but never a huge fan of polluting the `render` function with these conditionals statements, just a tad bit cleaner to have a dedicated component (e.g. `PluginIcon`, while renaming the other PluginIcon as Icon) that encapsulates this logic, but for now i think it's fine and we can fix this as part of another refactor